notebooks/basics/multiple-systems.ipynb

```diff
@@ -350,13 +350,53 @@
     "  \\alpha_m \\beta_1\\\\\n",
     "  \\vdots\\\\\n",
     "  \\alpha_m \\beta_k\n",
-    "  \\end{pmatrix}.\n",
+    "  \\end{pmatrix}\n",
+    "  \\tag{6}\n",
+    "$$\n",
+    "\n",
+    "This specification is known as the *Kronecker product* of two column vectors. The Kronecker product is simply the tensor product when applied to column vectors whose standard basis vectors are ordered alphabetically. We will usually refer to $(6)$ simply as the tensor product, but it is helpful to be familiar with the vocabulary. \n",
```

Comment:
Initially I had mentioned the Kronecker product, but I prefer not to open this can of worms... and I think it's better not to introduce new terminology that we won't use.